com.cubrid.common.update.feature/feature.properties

```diff
@@ -1,7 +1,7 @@
 featureName=CUBRID Tools Common Update Features
 providerName=Search Solution Corporation
 description=CUBRID Tools Common Update Feature
-copyright=Copyright (C) 2014 Search Solution Corporation. All rights reserved by Search Solution.
+copyright=Copyright (C) 2016 CUBRID Corporation. All rights reserved by Search Solution.
```

Comment:
Shouldn't it be changed to `All rights reserved by CUBRID.`? or how about changing the whole sentence to `Copyright (C) 2016 CUBRID Corporation all rights reserved.`?

Reply:
fix missing text in next commit. thanks